hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/map/SqlJsonTest.java

```diff
@@ -29,114 +28,103 @@
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.Map;
 
-import static com.hazelcast.jet.core.TestUtil.createMap;
+import static com.hazelcast.jet.pipeline.file.JsonFileFormat.FORMAT_JSON;
 import static com.hazelcast.jet.sql.impl.connector.SqlConnector.JAVA_FORMAT;
 import static com.hazelcast.jet.sql.impl.connector.SqlConnector.JSON_FLAT_FORMAT;
 import static com.hazelcast.jet.sql.impl.connector.SqlConnector.OPTION_KEY_CLASS;
 import static com.hazelcast.jet.sql.impl.connector.SqlConnector.OPTION_KEY_FORMAT;
 import static com.hazelcast.jet.sql.impl.connector.SqlConnector.OPTION_VALUE_FORMAT;
-import static java.lang.String.format;
+import static com.hazelcast.spi.properties.ClusterProperty.SQL_CUSTOM_TYPES_ENABLED;
 import static java.time.ZoneOffset.UTC;
-import static java.util.Arrays.asList;
-import static java.util.Collections.singletonList;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 public class SqlJsonTest extends SqlTestSupport {
-
     private static SqlService sqlService;
 
     @BeforeClass
-    public static void setUpClass() {
-        initialize(1, null);
+    public static void setup() {
+        initialize(1, smallInstanceConfig().setProperty(SQL_CUSTOM_TYPES_ENABLED.getName(), "true"));
```

Comment:
`SQL_CUSTOM_TYPES_ENABLED` shouldn't be needed for these tests and tests pass without it, similar in other cases

Reply:
Yes, it's not needed in `kafka/SqlJsonTest` as well.